src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedFrameworkReference.xaml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="ResolvedFrameworkReference"
+      Description="Framework Reference Properties"
+      DisplayName="Framework Reference"
+      PageTemplate="generic"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+  
+  <Rule.DataSource>
+    <DataSource HasConfigurationCondition="False"
+                ItemType="FrameworkReference"
+                MSBuildTarget="ResolveFrameworkReferencesDesignTime"
+                Persistence="ResolvedReference"
+                SourceOfDefaultValue="AfterContext"
+                SourceType="TargetResults" />
+  </Rule.DataSource>
+
+  <StringProperty Name="OriginalItemSpec"
+                  ReadOnly="True"
+                  Visible="False" />
+
+  <StringProperty Name="TargetingPackPath"
+                  DisplayName="Targeting Pack Path"
+                  ReadOnly="True" />
+
+  <StringProperty Name="TargetingPackName"
+                  DisplayName="Targeting Pack Name"
+                  ReadOnly="True" />
+
+  <StringProperty Name="TargetingPackVersion"
+                  DisplayName="Targeting Pack Version"
+                  ReadOnly="True" />
+
+  <StringProperty Name="Profile"
+                  DisplayName="Profile"
+                  ReadOnly="True" />
+
+  <StringProperty Name="RuntimePackPath"
+                  DisplayName="Runtime Pack Path"
+                  ReadOnly="True" />
+
+  <StringProperty Name="RuntimePackName"
+                  DisplayName="Runtime Pack Name"
+                  ReadOnly="True" />
+
+  <StringProperty Name="RuntimePackVersion"
+                  DisplayName="Runtime Pack Version"
+                  ReadOnly="True" />
+
+  <StringProperty Name="IsImplicitlyDefined"
```

Comment:
We're expecting both of these properties (IsImplicitlyDefined/PrivateAssets) to be copied to the ResolvedFrameworkReference, but they aren't currently: https://github.com/dotnet/sdk/pull/3355/files#r297966595.

Reply:
https://github.com/dotnet/sdk/pull/3355 was updated to include `IsImplicitlyDefined` but not `PrivateAssets`. It seems like your conversation with @dsplaisted was unfinished in this regard, and I'm not sure what the better path forward here is. I'll make a note on #4762 to track this.